### DISCUSSION TITLE
Sub/superscript

### DISCUSSION THREAD

**Discussion by HAF3** [standardnotes/forum#2556] [General] [answered]
Curious if anyone knew if any of the available editors support typing subscript and superscript.

Thanks

**Comment by JaspalSuri** [comment]
Hi @HAF3,

The [Bold](https://standardnotes.org/extensions/bold-editor), [Plus](https://standardnotes.org/extensions/plus-editor), and Monaco editors (within the [Append Editor](https://appendeditor.com)) support HTML syntax. Using the `sub` and `sup` tags will allow you to format the enclosed text subscripts and superscripts respectively.

**Comment by myreli** [comment, marked as answer]
Besides the ones mentioned by @JaspalSuri we also have this ability with the native [Super Editor](https://standardnotes.com/blog/introducing-super-notes-moments-and-offline-file-access):

<img width="1359" alt="image" src="https://user-images.githubusercontent.com/17554234/218790958-7bf7c5b6-e59a-419f-8767-7cb1fec61d55.png">
